### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by runesoerensen**
Not sure where/if we agreed on a place to put the Procfile buildpack and cnb-builder links?

Added a couple references here so we don't forget to include them, feel free to edit :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 ## What are Heroku Cloud Native Buildpacks?
 
-Heroku Cloud Native Buildpacks can help you create a production ready container image for your app with a single command, without the need to write a Dockerfile or configuring a complex build system. 
+Heroku Cloud Native Buildpacks, now in preview, can help you create a production ready container image for your app with a single command, without the need to write a Dockerfile or configuring a complex build system.
```
Comment (runesoerensen) [inline README.md:8]: Might make more sense to just include "preview" in the top header (e.g. "Heroku Cloud Native Buildpacks preview")?

File: README.md [hunk 2]
```diff
@@ -82,6 +84,14 @@ Each supported ecosystem has its own Git repository where all related buildpacks
 Heroku buildpacks are implemented in Rust. We developed and open-sourced a framework for writing Cloud Native Buildpacks in Rust called libcnb.rs.
 You can find [the repository on GitHub](https://github.com/heroku/libcnb.rs).
 
+### Builder Images
+
+A CNB builder image is a packaged set of buildpacks, base images and a lifecycle binary that orchestrates the build process. You can find [Heroku's CNB Builder Images here](https://github.com/heroku/cnb-builder-images).
+
```
Comment (edmorley) [inline README.md:90]: I wonder if we should include https://github.com/heroku/base-images too?

Comment (runesoerensen) [reply README.md:90]: As a separate "Associated Project", or just as a link under "Builder Images" when referring to [base images]( https://github.com/heroku/base-images)?

File: README.md [hunk 3]
```diff
@@ -82,6 +84,14 @@ Each supported ecosystem has its own Git repository where all related buildpacks
 Heroku buildpacks are implemented in Rust. We developed and open-sourced a framework for writing Cloud Native Buildpacks in Rust called libcnb.rs.
 You can find [the repository on GitHub](https://github.com/heroku/libcnb.rs).
 
+### Builder Images
+
+A CNB builder image is a packaged set of buildpacks, base images and a lifecycle binary that orchestrates the build process. You can find [Heroku's CNB Builder Images here](https://github.com/heroku/cnb-builder-images).
```
Comment (runesoerensen) [inline README.md:89]:
```suggestion
A CNB builder image is a packaged set of buildpacks, [base images](https://github.com/heroku/base-images) and a lifecycle binary that orchestrates the build process. You can find [Heroku's CNB Builder Images here](https://github.com/heroku/cnb-builder-images).
```

File: README.md (hunk 2 above)
Comment (runesoerensen) [reply README.md:90]: @edmorley I've just included it as a link when referring to it in the builder images section for now https://github.com/heroku/buildpacks/pull/7/commits/6426e87374c7a3383309abd0b6cb0e433b760e9f

File: README.md [hunk 4]
```diff
@@ -82,6 +84,14 @@ Each supported ecosystem has its own Git repository where all related buildpacks
 Heroku buildpacks are implemented in Rust. We developed and open-sourced a framework for writing Cloud Native Buildpacks in Rust called libcnb.rs.
 You can find [the repository on GitHub](https://github.com/heroku/libcnb.rs).
 
+### Builder Images
+
+A CNB builder image is a packaged set of buildpacks, [base images](https://github.com/heroku/base-images) and a lifecycle binary that orchestrates the build process. You can find [Heroku's CNB Builder Images here](https://github.com/heroku/cnb-builder-images).
+
+### Procfile Buildpack
+
+Our Cloud Native Procfile Buildpack is available [here on GitHub as well](https://github.com/heroku/buildpacks-procfile).
```
Comment (runesoerensen) [inline README.md:93]:
```suggestion
Our Procfile Cloud Native Buildpack is available [here on GitHub as well](https://github.com/heroku/buildpacks-procfile).
```

File: README.md [hunk 5]
```diff
@@ -53,7 +55,7 @@ pack build my-app-image --builder heroku/builder:22 --path node-js-getting-start
 
 ### Tutorial
 
-We also have a more in-depth tutorial available. It can be found in the [`TUTORIAL.md` file](TUTORIAL.md).
+Check out our [hands-on tutorial](TUTORIAL.md) to learn more about how our buildpacks can help you create production ready container images.
```
Comment (edmorley) [inline README.md:58]:
```suggestion
Check out our [hands-on tutorial](TUTORIAL.md) to learn more about how our buildpacks can help you create production-ready container images.
```

**Review by edmorley — APPROVED** [review]
Thank you :-)

**Pull request merged**